packages/react/src/ActionList/ActionList.figma.tsx

```diff
@@ -22,6 +29,73 @@ figma.connect(
   },
 )
 
+figma.connect(
+  ActionList.LeadingVisual,
+  'https://www.figma.com/design/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=15039-46399&t=YefUJRQj0NNo9Plz-4',
+  {
+    props: {
+      child: figma.children('*'),
+    },
+    example: ({child}) => <ActionList.LeadingVisual>{child}</ActionList.LeadingVisual>,
+  },
+)
+
+figma.connect(
+  ActionList.LeadingVisual,
+  'https://www.figma.com/design/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=15039-46400&t=YefUJRQj0NNo9Plz-4',
+  {
+    props: {
+      child: figma.children('*'),
+    },
+    example: ({child}) => <ActionList.LeadingVisual>{child}</ActionList.LeadingVisual>,
+  },
+)
+
+figma.connect(
+  ActionList.LeadingVisual,
+  'https://www.figma.com/design/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=23595-96989&t=YefUJRQj0NNo9Plz-4',
+  {
+    example: () => (
+      <ActionList.LeadingVisual>
+        <div style={{borderRadius: '24px', display: 'inline-block', height: '10px', width: '10px'}}></div>
```

Comment:
set the border-radius to 24px, wasn't sure what the var(--borderRadius-full) token mapped to in React @lukasoppermann

Reply:
Hey @francinelucca I think this should be reverted to `var(--borderRadius-full)`. This is available in dotCom and is the correct variable to use on anything that should be fully rounded.